com.unity.multiplayer.mlapi/Tests/Runtime/Messaging/UnnamedMessageTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections;
+using System.IO;
+using System.Text;
+using MLAPI.Serialization;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace MLAPI.RuntimeTests.Messaging
+{
+    public class UnnamedMessageTests : BaseMultiInstanceTest
```

Comment:
@ShadauxCat Added an additional test for unnamed messages, and made a similar fix for the stream position.

Same symptom for the resulting stream though, some trailing bytes.